build/ci/evergreen.yml

```diff
@@ -386,7 +386,7 @@ tasks:
           E2E_TAGS: atlas,config
   # If your e2e tests depend on a cluster running please consider setting it on its own task
   - name: atlas_generic_e2e
-    tags: ["e2e","generic"]
+    tags: ["e2e","generic","atlas"]
```

Comment:
[q] How was this working before?

Reply:
it's duplicating if you check the waterfall atlas is running om tests and om is running atlas
hard to spot on the PR as we took them out of the PR suite and you'll only see it in the waterfall